pygmt/src/grdfilter.py

```diff
@@ -122,8 +125,8 @@ def grdfilter(
     --------
     >>> from pathlib import Path
     >>> import pygmt
-    >>> # Apply a filter of 600 km (full width) to the @earth_relief_30m_g file
-    >>> # and return a filtered field (saved as netCDF)
+    >>> # Apply a filter of 600 km (full width) to the @earth_relief_30m_g file and
+    >>> # return a filtered field (saved as netCDF)
```

Comment:
```suggestion
    >>> # Apply a median filter of 600 km (full width) to the @earth_relief_30m_g grid
    >>> # and return a filtered grid (saved as netCDF file).
```

Not sure what "full width" means. The [GMT docs for **D** of `grdfilter`](https://docs.generic-mapping-tools.org/dev/grdfilter.html#d) say for `D=4` (`distance=4`):

> flag = 4: grid (x, y) in degrees, width in km, Spherical distance calculation.